core/src/main/java/io/questdb/cutlass/http/HttpServer.java

```diff
@@ -146,6 +146,18 @@ public HttpRequestProcessor newInstance() {
             });
         }
 
+        server.bind(new HttpRequestProcessorFactory() {
+            @Override
+            public String getUrl() {
+                return "/settings";
+            }
+
+            @Override
+            public HttpRequestProcessor newInstance() {
+                return new SettingsProcessor(cairoEngine.getConfiguration());
```

Comment:
nit: there is no state in SettingsProcessor, it can be singleton created once per server. Json inside it can also be re-computed.

Reply:
i have tried to recompute the json.
i will have to pkay with StringSink to make it work.
it escapes double-quotes, will change it.

Reply:
i will check if we create more than one instance of the processor.
agree, one should be enough.

Reply:
i am thinking that there might be a good reason why HttpResponseSink escapes double quotes.
maybe at the other end of the wire (in javascript) those escaped double quotes will appear just fine, and i should not worry about it.
so basically we have to send: `{\"oauth2.enabled\":\"true\"}`, and on the other end this will look like a proper json object:  `{"oauth2.enabled":"true"}`.
for example, in other tests i can see this:
```
testHttpClient.assertGet("{\"ddl\":\"OK\"}", "create table tab (value int, when long, ts timestamp) timestamp(ts) partition by day bypass wal;");
```
it expects `"{\"ddl\":\"OK\"}"`.
i will just use HttpResponseSink as is, i think it should be ok.

Reply:
nope, i was wrong... :)
it is not good to escape double quotes.
this will need some investigation why we do it.

Reply:
done.
very weird, unpredictable behaviour but if you do the Utf8 conversion yourself, and you pass in Utf8CharSequence,  HttpResponseSink will not escape double quotes.

also changed HttpServer to create only a single SettingsProcessor.